site/content/integrate/incoming-webhooks/_index.md

```diff
@@ -64,7 +64,7 @@ Incoming webhooks support more than just the `text` field. Here is a full list o
 | channel | Overrides the channel the message posts in. Use the channel's name and not the display name, e.g. use `town-square`, not `Town Square`.<br> Use an "@" followed by a username to send to a direct message.<br> Defaults to the channel set during webhook creation.<br> The webhook can post to any public channel and private channel the webhook creator is in.<br> Posts to direct messages will appear in the DM between the targeted user and the webhook creator. | No |
 | username | Overrides the username the message posts as.<br> Defaults to the username set during webhook creation or the webhook creator's username if the former was not set.<br> Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames). | No |
 | icon\_url | Overrides the profile picture the message posts with.<br> Defaults to the URL set during webhook creation or the webhook creator's profile picture if the former was not set.<br> Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons). | No |
-| icon\_emoji | Overrides the profile picture and `icon_url` parameter. Defaults to none and is not set during webhook creation. Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons). The expected content is an emoji name, as typed in a message but without `:`. | No |
+| icon\_emoji | Overrides the profile picture and `icon_url` parameter. Defaults to none and is not set during webhook creation. Must be enabled [in the configuration](https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-icons). The expected content is an emoji name, as typed in a message, with or without `:`. | No |
 | attachments | [Message attachments](https://docs.mattermost.com/developer/message-attachments.html) used for richer formatting options. | If `text` is not set, yes |
 | type | Sets the post `type`, mainly for use by plugins.<br> If not blank, must begin with "custom\_". | No |
 | props | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br> Mainly used by other integrations accessing posts through the REST API.<br> The following keys are reserved: "from\_webhook", "override\_username", "override\_icon\_url", "override\_icon\_emoji", "webhook\_display\_name", "card" and "attachments".    Props `card` allows for extra information (markdown formatted text) to be sent to Mattermost that will only be displayed in the RHS panel after a user clicks on an 'info' icon displayed alongside the post.  The info icon cannot be customized and is only rendered visible to the user if there is `card` data passed into the message. This is only available in v5.14+. There is currently no Mobile support for `card` functionality.  | No |
```

Comment:
```suggestion
| props | Sets the post `props`, a JSON property bag for storing extra or meta data on the post.<br> Mainly used by other integrations accessing posts through the REST API.<br> The following keys are reserved: "from\_webhook", "override\_username", "override\_icon\_url", "override\_icon\_emoji", "webhook\_display\_name", "card" ,and "attachments".<br> Props `card` allows for extra information (markdown formatted text) to be sent to Mattermost that will only be displayed in the RHS panel after a user clicks on an 'info' icon displayed alongside the post.<br> The info icon cannot be customized and is only rendered visible to the user if there is `card` data passed into the message.<br>  This is only available in v5.14+. <br> There is currently no Mobile support for `card` functionality. | No |
```

Reply:
I removed extra spacing and added <br> in case that spacing was meant to create breaks. I can't tell how it renders until the change is committed though.

Reply:
Tested locally, and looks good thanks!